packages/components/page_template/src/page_content.tsx

```diff
@@ -1,14 +1,33 @@
 import { Box, BoxProps } from "@buoysoftware/anchor-layout";
+import { LoadingIndicator } from "@buoysoftware/anchor-loading-indicator";
 
-type PageContentProps = Omit<BoxProps, "gridArea">;
+interface OwnProps {
+  loading?: boolean;
+}
+
+type PageContentProps = OwnProps & Omit<BoxProps, "gridArea">;
 
 export const PageContent: React.FC<PageContentProps> = ({
   children,
+  loading,
   ...props
 }): React.ReactElement => {
   return (
-    <Box gridArea="content" mt="40px" px="page.gutterX" {...props}>
-      {children}
+    <Box
+      gridArea="content"
+      display={loading ? "grid" : "block"}
```

Comment:
Should this be: 
```suggestion
      display={loading ? "flex" : "block"}
```

Reply:
Both work, not sure if one is better than the other.

Reply:
Actually, turns out if I use `display: flex` then `justify-self` doesn't work on the box below.

Per MDN:

> In flexbox layouts, this property is ignored. Read more about [alignment in Flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Alignment/Box_Alignment_in_Flexbox).